odoo/addons/base/ir/ir_ui_view.py

```diff
@@ -225,9 +225,13 @@ def replacer(m):
             if 'xml' in config['dev_mode'] and view.arch_fs and view.xml_id:
                 # It is safe to split on / herebelow because arch_fs is explicitely stored with '/'
                 fullpath = get_resource_path(*view.arch_fs.split('/'))
-                arch_fs = get_view_arch_from_file(fullpath, view.xml_id)
-                # replace %(xml_id)s, %(xml_id)d, %%(xml_id)s, %%(xml_id)d by the res_id
-                arch_fs = arch_fs and resolve_external_ids(arch_fs, view.xml_id)
+                if fullpath:
```

Comment:
I think that the commit msg is clear:  "fix dev mode when view is no more in addons-path"